backend/editor/entries.py

```diff
@@ -120,11 +121,26 @@ async def get_and_parse_taxonomy(self, uploadfile: UploadFile | None = None):
         try:
             with tempfile.TemporaryDirectory(prefix="taxonomy-") as tmpdir:
                 filepath = await (
-                    self.get_github_taxonomy_file(tmpdir)
+                    self.get_github_taxonomy_file(tmpdir, self.taxonomy_name)
                     if uploadfile is None
                     else self.get_local_taxonomy_file(tmpdir, uploadfile)
                 )
-                await run_in_threadpool(self.parse_taxonomy, filepath)
+                other_filepaths = None
+                if self.taxonomy_name == "ingredients":
+                    other_filepaths = []
+                    for external_taxonomy in (
+                        "additives_classes",
+                        "additives",
+                        "minerals",
+                        "vitamins",
+                        "nucleotides",
+                        "other_nutritional_substances",
+                    ):
+                        external_filepath = await self.get_github_taxonomy_file(
+                            tmpdir, external_taxonomy
+                        )
+                        other_filepaths.append(external_filepath)
```

Comment:
Could we put the list of external taxonomies in a configuration file instead ?

Maybe for now in settings.py 
```python
EXTERNAL_TAXONOMIES = {
    "ingredients": [
        "additives_classes",
        "additives",
        "minerals",
        "vitamins",
        "nucleotides",
        "other_nutritional_substances",
    ],
}
```

Reply:
Would it be interesting to even put it at the top of the `ingredients.txt` file under a `external_taxonomies` entry for example?

Reply:
Yep ! That would be a good idea too. We can make a PR to openfoodfacts-server, it's easy to accept such a PR.